rules/prefer-query-selector.js

```diff
@@ -106,9 +106,6 @@ const create = context => {
 		[selector](node) {
 			const method = node.callee.property.name;
 			const preferedSelector = forbiddenIdentifierNames.get(method);
```

Comment:
Unrelated, but I noticed a typo here.
```suggestion
			const preferredSelector = forbiddenIdentifierNames.get(method);
```